src/apps/lwaftr/icmp.lua

```diff
@@ -93,6 +94,22 @@ function new_icmpv4_packet(from_eth, to_eth, from_ip, to_ip, initial_pkt, l2_siz
    return new_pkt
 end
 
+function is_icmpv4(l2_size, pkt)
+   assert(l2_size > 2)
```

Comment:
It is absolutely unacceptable to use assert here, other than as a debug aid. Assert in a situation like this cannot ever be merged. Bringing down the whole lwaftr rather than returning false is not ok. Asserts can be done in self tests and basically nowhere else, aside from very temporarily in branches that are not ready to merge into our main dev branch.

Also, we've switched back to using the the ethernet header size again rather than l2_size in this type of context, thanks to @wingo 's vlan app work.